---
permalink: /about/
title: "About"
---
While the details of my professional experience can be found on my [LinkedIn](https://www.linkedin.com/in/amy-paguirigan-75a7299/) profile, here I'll provide some additional context. 

## Research Projects
### Functional Consequences of Mutations in Leukemia
- **Senior/Staff Scientist:**  11/2013 - present  
- Lead research using microfluidic device development and bioinformatic workflow design/optimization for single cell genotyping as well as bulk intratumoral genetic heterogeneity analyses (targeted DNA sequencing up to exome level); developed pipelines for microfulidic image analysis in R. 
- Received a K01 award (2015-2021) from the NIH's Big Data to Knowledge Initiative to support additional training to develop and improve ability to effectively manage, manipulate and analyze big datasets with a primary focus on genomics and bioinformatics.  Experience ranged from single cell transcriptomics to single cell genomics, SNP and gene expression arrays, targeted and exome-level DNA sequencing and bulk RNA sequencing analyses.  
- Developed R, python, bash, skills including using API’s, data visualization, Shiny app development, workflow managers, and reproducible use of bioinformatic software, extensive use of GitHub and documentation work.  
- Lead and performed laboratory and computational research into the integration of novel biotechnological techniques for single cell analysis and large scale bulk sequencing to clinical studies of clonal evolution in leukemia.


### Targeting Genotypes and Gene Expression in Single Cells
- **Postdoctoral Researcher:**  6/2008 – 10/2013; *Funding:* Fellow, American Cancer Society  
- Optimized and validated a range of fundamental protocols for use with single cells after such as multiplexed QPCR and genotyping to accurately study multiple aspects of cellular function in individual cells concurrently.  Developed and applied microfluidic devices for handling and isolating single cells or microvolume PCR reactions. Interfaced with CLIA laboratory to optimize approaches such that they could be applied to clinical specimens.  
- Obtained additional experience in molecular biology laboratory techniques, qPCR/digital PCR/highly multiplexed PCR, next generation sequencing (primarily Illumina), tissue culture, laboratory material and personnel management, project management, grant writing, project planning. 

### *In Situ* Protein Assays in Microfluidic Devices  
- **Graduate Fellow:** 5/2003-5/2008; *Funding:* Fellow, Department of Defense Breast Cancer Research Program; Fellow, Whitaker Foundation; Trainee, Computation and Informatics in Biology and Medicine Traineeship
- Adapted Western blotting techniques to quantify protein expression (in-cell Western) in microfluidic cultures via in situ infrared and visible antibody imaging (LICOR Odyssey/GE Typhoon).
- Biological assay development and application to microfluidic culture to determine the cellular baseline for a wide variety of stress responses; validation of a high throughput microfluidic culture platform using cell lines and primary cells using passive pumping for loading.
- Skills included: Multicolor confocal imaging, multicolor laser scanner analysis, microfabrication, Western blotting, tissue culture, primary cell culture.

## Technical Consulting  
During my time as a Staff Scientist, I have both learned from my mistakes and also shared that expertise with others who, by nature of their research, have similar needs and challenges.  I have been involved in a number of collaborations and projects where I served as a consultant to support groups in their training, career development and improvements in overall data stewardship chops.  

I currently support multiple projects for investigators at the Fred Hutch, providing subject matter expertise on data management for genomics data, advising and technical support for various cloud resources and guidance and training for software sharing.  I often advise and train staff in the use of GitHub, Docker, WDL based workflows, and reproducible code and workflow sharing, depending on the particular expertise level and needs of individual groups. 


### Translational Genomics Repository  
As the *Principal Investigator* of the Fred Hutch Translational Genomics Repository, I have developed a collaborative data management and analysis system supporting ongoing genomics research.  The project focuses on the use of on-prem and cloud (AWS, Google, and now Azure) based data storage, harmonized collection of associated clinical and scientific metadata across groups, and the use of workflow managers (Cromwell/Nextflow) plus containerization for reproducible bioinformatic analyses using both on-prem (SLURM) and cloud (AWS Batch) based computing resources.  End users included researchers with a range of experience with bioinformatics, large data management, and computing familiarity.   

- Build R packages and R Shiny applications facilitating expert/non-expert use of compute resources via workflow manager (Cromwell) API as well as straightforward access to processed datasets for analysis.  Basic use of python for supporting infrastructure/interactions with AWS.  
- Created the data management structure that supported cloud-based (primarily AWS S3) genomic data storage (including indexing and tagging of related datasets) of 1500+ research datasets and managed the associated clinical, experimental and laboratory metadata (including ontology development and harmonization across projects) across 8 collaborative projects and across 5 investigators.
- Curate a harmonized clinical, experimental and laboratory data ontology for data from human specimens (lung, multiple types of leukemia, myelodyplastic syndrome, normal donations) as well as for datasets from 4 other species, for a range of genomic data types from single cell RNA sequencing on up to whole exome sequencing datasets.  
- Interact closely with Fred Hutch IT, Business Development, Information Security Office and Institutional Review Board to ensure that technological solutions met requirements associated with human clinical and genomic data with respect to patient consent, study approval (by IRB), de-identification, access logging and all relevant security concerns.  
- Lead the testing and custom configuration of the workflow manager [Cromwell](https://github.com/broadinstitute/cromwell), for use by investigators at the Hutch, as well as write and share my own analysis workflows using Docker and WDL workflow language. You can read more about this work on my [Current Projects](/projects/) page. 


###  SciWiki
Another way in which I've shared what I've learned over the years is via a handful of community oriented efforts.  One of which is that I lead the [Fred Hutch Biomedical Data Science Wiki](https://sciwiki.fredhutch.org/) project, a collaborative documentation and training effort you can read more about on my [Current Projects](/projects/) page. 


### Data Strategist
I served as a **Data Strategist for the Hutch Data Commonwealth** at the Fred Hutch (9/2016-2/2018), advising with a focus on genomics data, primarily via the evaluation of multiple solutions for deploying bioinformatic workflows to cloud based computing resources, including the evaluation of multiple workflow manager software options in conjunction with Fred Hutch IT. 


## Education  
- Doctor of Philosophy in Biomedical Engineering, 2008; University of Wisconsin, Madison, WI
  - Concentration:  Microfluidic System Design, Optimization of Biological Assays, Cancer Biology
- Master of Science in Biomedical Engineering, 2005; University of Wisconsin, Madison, WI
  - Concentration: Modeling of Dynamic Biological Systems, Biopolymer/Polymer Science
- Bachelor of Science in Mechanical Engineering, 2003; University of Illinois, Urbana, IL
  - Concentration: Flow Cytometry, Biofluid Dynamics, Microhemodynamics
- Bachelor of Science in Biology, 2003; University of Illinois, Urbana, IL
  - Concentration: Bioengineering and Biophysics  


## Academic Publications  
Full publication list can be obtained via [NCBI MyBibliography](https://www.ncbi.nlm.nih.gov/myncbi/1pmiR_xKWbaQP/bibliography/public/)).

