---
layout: base
permalink: /research/
---
## Research Overview

A publication list can be obtained via [NCBI MyBibliography](https://www.ncbi.nlm.nih.gov/myncbi/1pmiR_xKWbaQP/bibliography/public/).

### Functional Consequences of Mutations in Leukemia
2012 - present - *Fred Hutchinson Cancer Research Center*
- Lead research using microfluidic device development and bioinformatic workflow design/optimization for single cell genotyping as well as bulk intratumoral genetic heterogeneity analyses (targeted DNA sequencing up to exome level); developed pipelines for microfluidic image analysis in R. 
- Received a K01 award (2015-2021) from the NIH's Big Data to Knowledge Initiative to support additional training to develop and improve ability to effectively manage, manipulate and analyze big datasets with a primary focus on genomics and bioinformatics.  Experience ranged from single cell transcriptomics to single cell genomics, SNP and gene expression arrays, targeted and exome-level DNA sequencing and bulk RNA sequencing analyses.  
- Developed R, python, bash, skills including using APIâ€™s, data visualization, Shiny app development, workflow managers, and reproducible use of bioinformatic software, extensive use of GitHub and documentation work.  
- Lead and performed laboratory and computational research into the integration of novel biotechnological techniques for single cell analysis and large scale bulk sequencing to clinical studies of clonal evolution in leukemia.
*Funding:* Primary funding through NIH/NCI funding via R01 CA226172 and R01 CA175215. 

### Single Cell and Intratumoral Heterogeneity Assay Development in Leukemia
2008 - 2012 - *Fred Hutchinson Cancer Research Center*
- Optimized and validated a range of fundamental protocols for use with single cells after such as multiplexed QPCR and genotyping to accurately study multiple aspects of cellular function in individual cells concurrently.  Developed and applied microfluidic devices for handling and isolating single cells or microvolume PCR reactions. Interfaced with CLIA laboratory to optimize approaches such that they could be applied to clinical specimens.  
- Obtained additional experience in molecular biology laboratory techniques, qPCR/digital PCR/highly multiplexed PCR, next generation sequencing (primarily Illumina), tissue culture, laboratory material and personnel management, project management, grant writing, project planning. 
*Funding:* Fellow, American Cancer Society 



### *In Situ* Protein Assays in Microfluidic Devices  
2003 - 2008 - *University of Wisconsin - Madison*
- Adapted Western blotting techniques to quantify protein expression (in-cell Western) in microfluidic cultures via in situ infrared and visible antibody imaging (LICOR Odyssey/GE Typhoon).
- Biological assay development and application to microfluidic culture to determine the cellular baseline for a wide variety of stress responses; validation of a high throughput microfluidic culture platform using cell lines and primary cells using passive pumping for loading.
- Skills included: Multicolor confocal imaging, multicolor laser scanner analysis, microfabrication, Western blotting, tissue culture, primary cell culture.
*Funding:* Fellow, Department of Defense Breast Cancer Research Program; Fellow, Whitaker Foundation; Trainee, Computation and Informatics in Biology and Medicine Traineeship 